content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -54,29 +54,42 @@ Clone the [DataTrails SCITT Examples](https://github.com/datatrails/datatrails-s
 1. Create a Python Virtual Environment for the sample scripts and install the dependencies
 
     ```bash
-    python -m  venv venv && \
+    python -m venv venv && \
     source venv/bin/activate && \
+    trap deactivate EXIT && \
     pip install --upgrade pip && \
     pip install -r requirements.txt
     ```
 
+      - **Note: If you receive errors**, delete the `venv` directory and try again:
+
+        ```bash
+        rm -r -f venv/
+        ```
+
 1. To ease copying and pasting commands, update any variables to fit your environment
 
     ```bash
     # your identity
     ISSUER="sample.synsation.io"
 
     # signing key to sign the SCITT Statements
-    SIGNING_KEY="my-signing-key.pem"
```

Comment:
Why /tmp ? that makes it a very unixy document. leaving it as plain file names is fine we just .gitignore the demo created files in the repo ?